code/modules/clothing/clothing_vr.dm

```diff
@@ -181,6 +181,13 @@
 		standing.layer = BODY_LAYER + 15 // 15 is above tail layer, so will not be covered by taurbody.
 	return standing
 
+/obj/item/clothing/suit/apply_accessories(var/image/standing)
+	if(LAZYLEN(accessories) && taurized)
+		for(var/obj/item/clothing/accessory/A in accessories)
+			var/image/I = new(A.get_mob_overlay())
+			I.pixel_x = 16 //Opposite of the pixel_x on the suit (-16) from taurization to cancel it out and puts the accessory in the correct place on the body.
+			standing.add_overlay(I)
+
```

Comment:
Needs an else return ..() at the end so the normal one runs if not taur. This is overriding another proc, and now all it does is taur ones. So if you don't have taur suit, it will get here, this if() will be false and then... Nothing. No accessories for you.

Reply:
Fffuuuuuck.